src/jsonapi-vuex.js

```diff
@@ -359,6 +359,37 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const addJvHelpers = (obj) => {
+  // Add Utility functions to _jv child object
+  Object.assign(obj[jvtag], {
+    is_rel(name) {
+      return name in obj[jvtag]['relationships']
+    },
+    is_attr(name) {
+      return name != '_jv' && !this.is_rel(name)
```

Comment:
Minor nit: if you don't need to do type coercion, it's better to use [!==](https://stackoverflow.com/a/359547)

Reply:
Agreed - will fix this.